docker-compose.yaml

```diff
@@ -123,6 +123,12 @@ services:
     - ./overlays/lava-common/usr/lib/python3/dist-packages/lava_common/schemas/boot/openocd.py:/usr/lib/python3/dist-packages/lava_common/schemas/boot/openocd.py:ro
     - ./overlays/lava-dispatcher/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/openocd.py:/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/openocd.py:ro
     - ./overlays/lava-dispatcher/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/strategies.py:/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/strategies.py:ro
+    - ./overlays/lava-dispatcher/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/jlink.py:/usr/lib/python3/dist-packages/lava_dispatcher/actions/boot/jlink.py:ro
+    - ./overlays/lava-dispatcher/usr/lib/python3/dist-packages/lava_dispatcher/tests/test_jlink.py:/usr/lib/python3/dist-packages/lava_dispatcher/tests/test_jlink.py:ro
+    - ./overlays/lava-dispatcher/usr/lib/python3/dist-packages/lava_dispatcher/tests/pipline_refs/jlink.yaml:/usr/lib/python3/dist-packages/lava_dispatcher/tests/pipline_refs/jlink.yaml:ro
```

Comment:
> pipline_refs

This shows why this approach doesn't scale. Let's concentrate on https://github.com/Linaro/lite-lava-docker-compose/pull/15/ instead (i.e. yes, you host your own fork of LAVA until it's merged upstream, and we need to combine a few WIP changes from different folks, then yeah, we need to maintain LITE staging branch).

cc: @galak, @vanti